temporian/implementation/numpy/operators/core_mapping.py

```diff
@@ -1,8 +1,10 @@
 from temporian.implementation.numpy.operators.select import NumpySelectOperator
 from temporian.implementation.numpy.operators.sum import NumpySumOperator
+from temporian.implementation.numpy.operators.assign import NumpyAssignOperator
```

Comment:
Wrong import order - `from temporian.implementation.numpy.operators.assign import NumpyAssignOperator` should go first.